Makefile

```diff
@@ -79,12 +79,16 @@ $(OUTPUT_DIR)/libp2p_port: $(PORT_SOURCES)
 
```

Comment:
should we add something in the README? In the installing and running part.

Reply:
Yes. Good catch!